2020-05-07 17:25:14,344 - D:/python/loging_test.py[line:38] - ERROR: error
